lib/registries/addon/overview/index/template.hbs

```diff
@@ -3,9 +3,9 @@
         {{t 'registries.overview.component_of'}}
         <OsfLink
             @route='resolve-guid'
-            @models={{array this.root.id}}
+            @models={{array this.root}}
         >
-            {{this.root.title}}
+            {{this.root}}
```

Comment:
This is not ideal, since GUIDs don't really mean that much to people. What happens if you revery out these changes and wrap the OsfLink in an `if` statement testing for `root`?

Reply:
Ok, this has now been updated to use the root value component name, not the guid value of the registeredFrom registration.

Reply:
Brian, I'm not sure the fix you are referring to. Can you share more info regarding this?